scripts/masternode.js

```diff
@@ -193,6 +206,22 @@ export default class Masternode {
         return [v + 27, ...signature];
     }
 
+    async getWalletPublicKey() {
+        if (masterKey.isHardwareWallet) {
+            const hi = await masterKey.getPublicKey(this.walletPrivateKeyPath);
+            console.log(hi);
+            return hexToBytes(hi);
```

Comment:
Nuke the `hi` variable which looks like it was only used for logging/debugging purposes.
```suggestion
           return hexToBytes(await masterKey.getPublicKey(this.walletPrivateKeyPath));
```